packs/feats/ancestral-blood-magic.json

```
@@ -24,7 +24,15 @@
             "remaster": false,
             "title": "Pathfinder Advanced Player's Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.Sorcerer.Bloodline.BloodMagicDescription.AncestralBloodMagic",
+                "option": "ancestral-blood-magic",
+                "placement": "spellcasting",
+                "toggleable": true
+            }
```

```suggestion
            {
                "key": "RollOption",
                "label": "PF2E.SpecificRule.Sorcerer.Bloodline.BloodMagicDescription.AncestralBloodMagic",
                "option": "item:tag:ancestral-spell",
                "placement": "spellcasting",
                "toggleable": true
            }
```
In theory this could be completely automated soon:tm:, so let's set ourselves up for success with the choice of roll option.